circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 5.3.0
```

Comment:
Why NodeJS 5.3.0? It's neither the 4.x.x LTS or 6.x.x Current release? https://nodejs.org/en/